src/gg.regression.unity.bots/Runtime/Scripts/ClientDashboard/RGTcpManager.cs

```diff
@@ -151,6 +177,12 @@ private void ProcessResolvedSequences(IDictionary<string, (string, BotSequence)>
             m_availableBotSequences = sequences.Select(kvp => new AvailableBotSequence(kvp.Key, kvp.Value.Item2)).ToList();
             SendAvailableSequences();
         }
+
+        private void LoadAvailableSegments()
```

Comment:
based on the method naming, I don't like how these methods implicitly send the status as well, maybe rename them to better represent that they do that ?

Reply:
Updated the name to indicate that this does send segments after loading them